learning/step_by_step/your_first_game.rst

```diff
@@ -0,0 +1,768 @@
+.. _doc_your_first_game:
+
+Your First Game
+===============
+
+Overview
+--------
+
+This tutorial will guide you through making your first Godot Engine
+project. You will learn how the Godot Engine editor works, how a project
+is structured, and how to build a 2D game.
+
+.. note:: This project is intended to introduce the Godot Engine. It 
+          assumes that you have some programming experience already. If 
+          you're new to programming entirely, you should start here:
+          :ref:`doc_scripting`.
+
+The game is called *"Dodge the Creeps"*. Your character must move and
+avoid the enemies for as long as possible. Here is a preview of the
+final result:
+
+.. image:: /img/dodge_preview.gif
+
+**Why 2D?** 
+    Beginners are urged to focus on 2D projects until they
```

Comment:
Answer the question straight away:
`3D games are a lot more complex than 2D ones. That's why you should stick to 2D until you have a good understanding of the game development process.`